### PR TITLE
fix: colletion fetch issue

### PR DESCRIPTION
**Pull request by theketan26**
## Description

On starting application or changing workspace, collection will be fetched from service

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [ ] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by punitkashyup** [comment]
Please attach the issue link

File: src/packages/@app/pages/Collections/CollectionPage.ViewModel.ts [hunk 1]
```diff
@@ -116,6 +116,15 @@ export default class CollectionsViewModel {
...
   constructor() {}
+  
+  public fetchCollections = async (workspaceId: string) => {
+    const res = await this.collectionService.fetchCollection(workspaceId);
+    if (res.status) {
+      this.collectionRepository.bulkInsertData(res.data.data);
+    } else {
+      notifications.error("Failed to fetch collections!")
+    }
+  }
```
Comment (itsmdasifraza) [inline src/packages/@app/pages/Collections/CollectionPage.ViewModel.ts:127]: add js-doc comments here

Comment (itsmdasifraza) [reply src/packages/@app/pages/Collections/CollectionPage.ViewModel.ts:127]: i think there will be res.isSuccessful

**Comment by punitkashyup** [comment]
Please attach the issue link

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**